lib/wanda_web/schemas/check_execution.ex

```diff
@@ -0,0 +1,20 @@
+defmodule WandaWeb.Schemas.CheckExecution do
+  @moduledoc """
+  Check execution API spec
+  """
+
+  require OpenApiSpex
+
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "Check execution",
+    description: "A single execution of checks",
+    type: :object,
+    properties: %{
+      execution_id: %Schema{type: :string, description: "Execution ID"},
```

Comment:
We could add a `format: :uuid` to `execution_id` and `group_id`, for a friendlier representation on the api doc.

Reply:
I tried but I got an error, i googled a lot but I didn't find a solution that wasn't "create your own format with a huge regex"

Reply:
Not talking about validation, no need for regex :smile: 

Just really a silly thing like we do here for example https://github.com/trento-project/web/blob/main/lib/trento_web/openapi/schema/host.ex#L37

```elixir
%Schema{type: :string, description: "Execution ID", format: :uuid},
```

It would take care of giving that information in the swagger ui and it generate uuid samples.
If it is causing troubles feel free to skip.